specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json

```diff
@@ -0,0 +1,1940 @@
+      "description": "Information about the extracted cell in a table.",
+      "type": "object",
+      "required": [
+        "rowIndex",
+        "columnIndex",
+        "text",
+        "boundingBox",
+        "confidence"
+      ],
+      "properties": {
+        "rowIndex": {
+          "description": "Row index of the cell.",
+          "type": "integer",
+          "minimum": 0,
+          "x-nullable": false
+        },
+        "columnIndex": {
+          "description": "Column index of the cell.",
+          "type": "integer",
+          "minimum": 0,
+          "x-nullable": false
+        },
+        "rowSpan": {
+          "description": "Number of rows spanned by this cell.",
+          "type": "integer",
+          "minimum": 1,
+          "default": 1,
+          "x-nullable": false
+        },
+        "columnSpan": {
+          "description": "Number of columns spanned by this cell.",
+          "type": "integer",
+          "minimum": 1,
+          "default": 1,
+          "x-nullable": false
+        },
+        "text": {
+          "description": "Text content of the cell.",
+          "type": "string"
+        },
+        "boundingBox": {
+          "description": "Bounding box of the cell.",
+          "$ref": "#/definitions/BoundingBox"
+        },
+        "confidence": {
+          "description": "Confidence value.",
+          "$ref": "#/definitions/Confidence"
+        },
+        "elements": {
+          "description": "When includeTextDetails is set to true, a list of references to the text elements constituting this table cell.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ElementReference"
+          }
+        },
+        "isHeader": {
+          "description": "Is the current cell a header cell?",
+          "type": "boolean",
+          "default": false,
+          "x-nullable": false
+        },
+        "isFooter": {
+          "description": "Is the current cell a footer cell?",
+          "type": "boolean",
+          "default": false,
+          "x-nullable": false
+        }
+      }
+    },
+    "DocumentResult": {
+      "description": "A set of extracted fields corresponding to the input document.",
+      "type": "object",
+      "required": [
+        "docType",
+        "pageRange",
+        "fields"
+      ],
+      "properties": {
+        "docType": {
+          "description": "Document type.",
+          "type": "string"
+        },
+        "modelId": {
+          "description": "Model identifier.",
+          "type": "string",
+          "format": "uuid",
+          "x-nullable": false
+        },
+        "pageRange": {
+          "description": "First and last page number where the document is found.",
+          "type": "array",
+          "minItems": 2,
+          "maxItems": 2,
+          "items": {
+            "type": "integer",
+            "minimum": 1,
+            "x-nullable": false
+          }
+        },
+        "docTypeConfidence": {
```

Comment:
Will this always be 1.0 for non-composed models?

Reply:
yes.

Reply:
To be more precise, this will be 1.0 when the form type is specified via the analysis operation (ex. prebuilt/receipt) or because there is only one model.  It may not be 1.0 when we are performing auto-detection.  This currently happens for composed models.  In the future, we may introduce formrecognizer/v3.0/analyze that can perform custom/prebuilt model auto-detection.